### PR TITLE
refactor: logs and toasts

### PR DESCRIPTION
**Pull request by RonenMars**
## Description
Go through the whole app, services, and other actions and validate that every action has it log.

## Linear Ticket
https://linear.app/autokitteh/issue/UI-256/services-validate-all-log-errors

## What type of PR is this? (check all applicable)

-   [ ] 💡 (feat) - A new feature (non-breaking change which adds functionality)
-   [ ] 🔄 (refactor) - Code Refactoring - A code change that neither fixes a bug nor adds a feature
-   [ ] 🐞 (fix) - Bug Fix (non-breaking change which fixes an issue)
-   [ ] 🏎 (perf) - Optimization
-   [ ] 📄 (docs) - Documentation - Documentation only changes
-   [ ] 📄 (test) - Tests - Adding missing tests or correcting existing tests
-   [ ] 🎨 (style) - Styles - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] ⚙️ (ci) - Continuous Integrations - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] ☑️ (chore) - Chores - Other changes that don't modify src or test files
-   [ ] ↩️ (revert) - Reverts - Reverts a previous commit(s).

### CODE REVIEW

File: src/services/triggers.service.ts [hunk 1]
```diff
@@ -80,11 +80,6 @@ export class TriggersService {
 			const { data: environments, error: errorEnvs } = await EnvironmentsService.listByProjectId(projectId);
 
 			if (errorEnvs) {
-				LoggerService.error(
```
Comment (J1za) [inline src/services/triggers.service.ts:83]: why you remove Loggger?

File: src/services/triggers.service.ts [hunk 2]
```diff
@@ -105,11 +100,6 @@ export class TriggersService {
 			const { data: environments, error } = await EnvironmentsService.listByProjectId(projectId);
 
 			if (error) {
-				LoggerService.error(
```
Comment (J1za) [inline src/services/triggers.service.ts:108]: don't we need the Logger?

File: src/services/environments.service.ts [hunk 3]
```diff
@@ -25,25 +30,30 @@ export class EnvironmentsService {
...
 			if (error) {
-				return { data: undefined, error };
+				const errorMessage = i18n.t("defaultEnvironmentNotFoundExtended", {
+					error: (error as Error).message,
+					projectId,
+					ns: "services",
+				});
+				LoggerService.error(namespaces.environmentsService, errorMessage);
+
+				return { data: undefined, error: errorMessage };
```
[the remark this answers is not in the document]
Comment (RonenMars) [reply src/services/environments.service.ts:40]: removed all except the return

File: src/services/triggers.service.ts (hunk 2 above)
Comment (RonenMars) [reply src/services/triggers.service.ts:108]: Same as above, logged in `EnvironmentsService.listByProjectId`

File: src/components/organisms/deployments/sessions/viewer.tsx [hunk 4]
```diff
@@ -46,20 +46,20 @@ export const SessionViewer = () => {
...
+
+			return;
+		}
+		if (!sessionInfoResponse) {
+			addToast({ message: t("sessionNotFound"), type: "error" });
+
+			return;
 		}
+		setSessionInfo(sessionInfoResponse);
+		setIsLoading(false);
```
[the remark this answers is not in the document]
Comment (RonenMars) [reply src/components/organisms/deployments/sessions/viewer.tsx]: refactored

**Review by J1za — APPROVED** [review]

**Pull request merged**